src/nginx/t/transcoding_deep_struct.t

```diff
@@ -113,12 +113,12 @@ $request
 EOF
 
 # 400 Bad request error coming from the backend as it fails to parse the proto so deep.
```

Comment:
500

Reply:
Will do. 

I am doing the same test on envoy grpc-json trancoding. For the valid deep struct, it returns 503 while ESP returns 500. I am thinking if it is a bug for esp, since the request body is a bool for echoStruct, the past request doesn't find the corresponding api. 

https://github.com/cloudendpoints/esp/blob/fc65c3e3ebafc2d6df3cfab12e002c5cd244b84f/test/transcoding/service.pb.txt#L140